.gitmodules

```diff
@@ -7,6 +7,9 @@
 [submodule "third_party/cpp-btree"]
 	path = third_party/cpp-btree
 	url = https://github.com/mrks/cpp-btree
+[submodule "third_party/compact_vector"]
+	path = third_party/compact_vector
+	url = https://github.com/benrobby/compact_vector.git
```

Comment:
Do we need to use "your version"? Might introduce some long term problems.

Reply:
I know, I know. They forgot to include some stl headers, so "my version" only adds two lines. I'll try their version again to see if the error is fixed by now and if not I can open a PR.

Reply:
works now, reverted back to the official version

Reply:
What do you mean? It‘s now gmarcais/compact_vector

Reply:
I just give up with GitHub. Nevermind, looks good.